auth0/resource_auth0_attack_protection.go

```diff
@@ -45,11 +48,12 @@ func newAttackProtection() *schema.Resource {
 									"admin_notification",
 								}, false),
 							},
-							Optional:    true,
 							Description: "Action to take when a breached password is detected.",
 						},
 						"admin_notification_frequency": {
-							Type: schema.TypeList,
+							Type:     schema.TypeSet,
+							Optional: true,
+							Computed: true,
```

Comment:
For all other occurences of TypeSet and Computed set to true please check above comments.